owl-bot-migration/tmpl/owlbot_py.tmpl

```diff
@@ -0,0 +1,41 @@
+# Copyright {{copyright_year}} Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import logging
+from pathlib import Path
+import subprocess
+
+import synthtool as s
+from synthtool.languages import php
+
+logging.basicConfig(level=logging.DEBUG)
+
+src = Path(f"../{php.STAGING_DIR}/{{target_dir}}").resolve()
+dest = Path().resolve()
+
+{% if copy_partial_veneer -%}
+php.owlbot_main(src=src, dest=dest)
+{% else -%}
+php.owlbot_main(
+    src=src,
+    dest=dest,
+    copy_excludes=[
+        src / "*/src/*/*.php"
```

Comment:
In the newer apis, we exclude the `proto/**/*_*.php` files as well... we will want to remember to add this. Those APIs are:

* DocumentAi
* Deploy
* ContactCenterInsights
* ContainerAnalysis
* Functions
* NetworkSecurity
* Grafeas

Reply:
We will also need this code for those APIs:
```php
# remove class_alias code
s.replace(
    "src/V*/*/*.php",
    r"^// Adding a class alias for backwards compatibility with the previous class name.$"
    + "\n"
    + r"^class_alias\(.*\);$"
    + "\n",
    '')
```

Reply:
If there's a mechanical way to distinguish this, I can add it. Otherwise, I'd do it manually

Reply:
Maybe we can add that list of APIs statically and do a special treatment.

Reply:
@bshaffer Added some code for handling newer APIs.

Reply:
looks good! thanks!